unittests/resource_limits_test.cpp

```diff
@@ -334,4 +335,84 @@ BOOST_AUTO_TEST_SUITE(resource_limits_test)
       add_transaction_usage( {N(dan)}, 34, 0, 2 + blocks_per_day );
    } FC_LOG_AND_RETHROW()
 
-BOOST_AUTO_TEST_SUITE_END()
+   /**
+     * Test to make sure that get_account_net_limit_ex and get_account_cpu_limit_ex returns proper results, including
+     * 1. the last updated timestamp is always same as the time slot on accumulator.
+     * 2. when no timestamp is given, the current_used should be same as used.
+     * 3. when timestamp is given, if it is earlier than last_updated_time_stamp, current_used is same as used,
+     *    otherwise, current_used should be the decay value (will be 0 after 1 day)
+    */
+   BOOST_FIXTURE_TEST_CASE(get_account_net_limit_ex_and_get_account_cpu_limit_ex, resource_limits_fixture) try {
+
+      const account_name cpu_test_account("cpuacc");
+      const account_name net_test_account("netacc");
+      const uint32_t delta_slot = 100;
```

Comment:
Use constexpr and add static_assert that `delta_slot` is less than `net_window` and `cpu_window` since the test relies on that assumption.